projects/packages/videopress/src/client/state/reducers.js

```diff
@@ -368,6 +370,54 @@ const videos = ( state, action ) => {
 			};
 		}
 
+		case SET_UPDATING_VIDEO_POSTER: {
+			const { id } = action;
+			const currentMeta = state?._meta || {};
+			const currentMetaItems = currentMeta?.items || {};
+
+			return {
+				...state,
+				_meta: {
+					...currentMeta,
+					items: {
+						...currentMetaItems,
+						[ id ]: {
```

Comment:
You should keep the current meta for this one, not? You're overriding it right now

Reply:
Good catch!